playback/Playback/PlaybackConstant.cs

```diff
@@ -0,0 +1,8 @@
+namespace Playback
+{
+    public static class PlayBackConstant
+    {
+        public static string ExtendedName = ".thuaipb";
+        public static byte[] Prefix = { (byte)'P', (byte)'B', 0, 0 };   // 文件前缀，用于标识文件类型，版本号为5
```

Comment:
```suggestion
        public static byte[] Prefix = { (byte)'P', (byte)'B', 6, 0 };   // 文件前缀，用于标识文件类型，版本号为 6
```
这里可以设置版本号为 `6` 了（使用小端序，3\~4 字节为 16 位版本号，所以 `6` 在第三个字节上）